Makefile

```diff
@@ -129,14 +129,22 @@ dev-run: $(GO_BIN)/CompileDaemon $(GO_BIN)/openfga ## Run the OpenFGA server wit
 			CompileDaemon -graceful-kill -build='make install' -command="openfga run --datastore-engine mysql --datastore-uri root:secret@tcp(localhost:3306)/openfga?parseTime=true"; \
 			break; \
 			;; \
-		"postgres") \
-			echo "==> Running OpenFGA with Postgres data storage"; \
+		"postgres14") \
```

Comment:
a bit concern that this is a backward breaking change.  Will it be possible to keep this as postgres? (or have both posgres and postgres14)

Reply:
yeah, I'll change `postgres17` to postgres so that there is no breaking change to scripts, but the behavior is changed to default to the newest pg